### ISSUE TITLE
Warning: Breach of contract warning letter

### ISSUE THREAD

**Issue by 106umao** [Kamandlou/fux#1]
@Kamandlou 

I don't understand why you would lie to me for $150, you are so poor that you are crazy? Besides, do you really think I only have one github account? I can have countless accounts if I want, and this is just one of them.
I have taken a snapshot of your watch list and followed list. I will use ten different email addresses to send you and anyone who has any contact with you every day. In addition, I will also use ten different email addresses every day. The newly registered github account will leave debt collection information under all your warehouses, 
In addition, this is true for all repositories that you are contributing to, have contributed to, and want to share.
To sum up, if you are willing to bear the price to cheat me $150, please feel free to do so. Anyway, I have automated procedures for doing this. It does not require any times on my part. It does not require any times on my part.If you are sensible, please return my money to me quickly!
